### PR TITLE
feat: add cors

### PR DESCRIPTION
**Pull request by XVincentX**
CORS Support

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -8,6 +8,13 @@
 
 module Todos
   class Application < Rails::Application
+
+    config.middleware.insert_before 0, "Rack::Cors" do
+      allow do
+        origins '*'
+        resource '*', headers: :any, methods: [:get, :post, :options]
```
Comment (marbemac) [inline config/application.rb]: Methods any? Deletes, puts, etc won't work with this right?

**Review by marbemac — CHANGES_REQUESTED** [review]

**Comment by marbemac** [comment]
Pretty sure this has not been deployed in like 3 years, kind of scared haha - we'll see!

File: config/application.rb (hunk 1 above)
Comment (XVincentX) [reply config/application.rb]: Just get, post and options; do you want me to change to any?

Comment (marbemac) [reply config/application.rb]: Yeah, no harm right? And todos api supports delete, patch, and put in any case.

**Pull request merged**